src/apis/BackendApi.ts

```diff
@@ -0,0 +1,23 @@
+import KapacitorEvent from '@/interfaces/KapacitorEvent';
+
+export default class BackendApi {
+    // region public members
+    // endregion
+
+    // region private members
+    // endregion
+
+    // region constructor
+    // endregion
+
+    // region public methods
+    public static async events(topicName: string): Promise<KapacitorEvent[]> {
+        const response = await fetch(`http://82.140.0.78:8082/events/${topicName}?min-level=OK`);
```

Comment:
Do we really want to show OK-Events in the detail view?

Reply:
Changed that to WARNING.